hbp_knowledge/hypoxia/zhang2011.bel

```diff
@@ -0,0 +1,654 @@
+neurons from hypoxia-induced cell death by binding with neuropilin-1, a receptor known to
+be involved in axon guidance during development [42, 45]."
+SET MeSHAnatomy = "Motor Neurons"
+bp(GO:"response to hypoxia") -> bp(GO:"cell death")
+complex(p(HGNC:VEGFA),p(HGNC:NRP1)) -| bp(GO:"cell death")
+p(HGNC:NRP1) -- bp(GO:"axon guidance")
+UNSET MeSHAnatomy
+
+SET Evidence = "It has
+been reported that EPO levels are significantly decreased in cerebrospinal fluids from
+patients with differing severity and duration of ALS [46], while EPO immunoreacitivity is
+significantly increased in midbrain, brain stem, and cortex of the SOD1G93A mutant mice
+[47]."
+SET MeSHAnatomy = "Cerebrospinal Fluid"
+path(DOID:"amyotrophic lateral sclerosis") -| p(HGNC:EPO)
+UNSET MeSHAnatomy
+SET Species = "10090"
+SET Anatomy = {"midbrain","brainstem","cortex"}
+p(HGNC:SOD1,var("Gly93Ala")) -> p(HGNC:EPO)
+UNSET Anatomy
+UNSET Species
+
+SET Evidence = "In this SOD1G93A mutant mouse ALS model, administration of EPO suppresses the
+onset and progression of ALS by preventing motor neuron death and inflammation [48–49]."
+SET Species = "10090"
+composite(p(HGNC:SOD1,var("Gly93Ala")),p(HGNC:EPO)) -| path(DOID:"amyotrophic lateral sclerosis")
+composite(p(HGNC:SOD1,var("Gly93Ala")),p(HGNC:EPO)) -| bp(GO:"neuron death")
+composite(p(HGNC:SOD1,var("Gly93Ala")),p(HGNC:EPO)) -| bp(GO:"inflammatory response")
+UNSET Species
+
+SET Evidence = "Indeed, the iron chelator M30, which up-regulates HIF-1 expression,
+protects NSC-34 motor neuron cells from oxidative damage in vitro and significantly delays
+the onset of ALS in SOD1G93A mutant mice [50] (Table 2)."
+#NSC-34 cell line
+a(PUBCHEM:6918822) pos p(MESH:"Hypoxia-Inducible Factor 1")
+a(PUBCHEM:6918822) -| bp(GO:"neuron death in response to oxidative stress")
+SET Species = "10090"
+composite(p(HGNC:SOD1,var("Gly93Ala")),a(PUBCHEM:6918822)) -| path(DOID:"amyotrophic lateral sclerosis")
+UNSET Species
+
+UNSET MeSHDisease
+
+SET MeSHDisease = "Alzheimer Disease"
+
+SET Evidence = "The HIF-1 inducer, M30, has been reported to increase the HIF-1α protein level and elevate
+expression of the HIF-1 target genes, VEGF and EPO."
+a(PUBCHEM:6918822) -> r(HGNC:HIF1A)
+a(PUBCHEM:6918822) -> r(HGNC:EPO)
+a(PUBCHEM:6918822) -> r(HGNC:VEGFA)
+
+SET Evidence = "It has been reported that M30
+simultaneously attenuates tau phosphorylation and protects cortical neurons against Aβ (25–
+35) toxicity [57]."
+a(PUBCHEM:6918822) -| p(HGNC:MAPT,pmod(Ph))
+SET Anatomy = "cortex"
+a(PUBCHEM:6918822) -| path(MESH:"Amyloid Neuropathies")
+UNSET Anatomy
+
+SET Evidence = "Overexpression of HIF-1α has been shown to protect the rat sympathetic
+nerves-like cell line PC12, the central nervous system cell line B12, and primary cultured
+cortical neurons from Aβ-induced neurotoxicity [58], possibly through activating glycolytic
+and hexose monophosphate shunt-related enzymes."
+SET CellLine = {"PC12","B12 cell"}
```

Comment:
the evidence that HIF1a protects from Abeta induced neurotoxicity is not encoded